### DISCUSSION TITLE
RenderTargetBitmap

### DISCUSSION THREAD

**Discussion by wkmhv** [unoplatform/uno#18016] [Q&A]
![grafik](https://github.com/user-attachments/assets/1622724c-6b80-4a0d-97c1-3a7883f0ee13)

Why does Visual Studio show that "RenderTargetBitmap" is not supported even though it works fine under net8.0-windows10.0.1941? Unfortunately, it does not work as expected under net8.0-android.

**Comment by morning4coffe-dev** [comment]
Hey @wkmhv,

This generally means the feature is not supported by Uno targets (the `net8.0-windows10.0.1941` is a pure WinUI from Microsoft and isn't managed by Uno).

Regarding `RenderTargetBitmap`, I'm not entirely sure what you mean by it not working as expected. Could you provide more details or reproduce the bug you're encountering? If you look at the [source code](https://github.com/unoplatform/uno/blob/1129f49e197bbd28a87991c04a08d3b2cac825e1/src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.cs), you'll see that the `RenderAsync` method you're using is not currently supported on other platforms. You can find more details for Android [here](https://github.com/unoplatform/uno/blob/1129f49e197bbd28a87991c04a08d3b2cac825e1/src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.Android.cs). Please let me know if this helps.

**Reply by MartinZikmund** [reply]
I made PR to adjust the NotImplemented reporting for the ctor, but please let us know with a repro on what is not working for you, as the API itself should work correctly on Android

**Reply by wkmhv** [reply]
it works now also on android. Visual Studio show that "RenderTargetBitmap" is not supported ?
[Uno5.zip](https://github.com/user-attachments/files/16758589/Uno5.zip)

**Reply by morning4coffe-dev** [reply]
@wkmhv I had a quick glance at your sample, and it seems that you are still using Uno.Sdk 5.3.99. Can you update to the latest preview to see if this behavior also gets fixed? The fixes should be available in the latest version, 5.4.0-dev.226 or higher. Can you please confirm?

**Reply by wkmhv** [reply]
![image](https://github.com/user-attachments/assets/d7856ad8-054e-4c9d-a2db-ab462f3d65e1)

![image](https://github.com/user-attachments/assets/1e5482c9-be26-4988-8282-8febc038338b)

**Reply by morning4coffe-dev** [reply]
@wkmhv For updating Uno.Sdk, you need to go to the Solution Items > global.json and update it from there (you might need to restart VS after you do so, it should prompt you)
![image](https://github.com/user-attachments/assets/1cd803e0-47cf-4ef4-971e-069f53871127)

You can find the latest versions on Nuget - https://www.nuget.org/packages/Uno.Sdk/
